frontend/src/metabase/query_builder/components/dataref/ConnectedTableList.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { jt } from "ttag";
+
+import Table from "metabase-lib/lib/metadata/Table";
+
+import {
+  NodeListItem,
+  NodeListItemName,
+  NodeListItemIcon,
+  NodeListTitle,
+  NodeListContainer,
+  NodeListIcon,
+  NodeListTitleText,
+} from "./NodeList.styled";
+
+type Props = {
```

Comment:
nit: We use `interface` instead of `type` for almost anything, including props. The naming convention is `<component-name>Props`, e.g. `ConnectedTableListProps`